charts/sysdig/templates/configmap-image-analyzer.yaml

```diff
@@ -55,7 +55,7 @@ metadata:
 {{ include "sysdig.labels" . | indent 4 }}
 data:
   {{- if .Values.nodeAnalyzer.collectorEndpoint }}
-  collector_endpoint: {{ .Values.nodeAnalyzer.collectorEndpoint }}
+  collector_endpoint: "https://{{ .Values.nodeAnalyzer.collectorEndpoint }}/internal/scanning/scanning-analysis-collector"
```

Comment:
If `.Values.nodeAnalyzer.collectorEndpoint = https://secure.sysdig.com` as per the readme change, won't this line evaluate to  `collector_endpoint: "https://https://secure.sysdig.com/internal/scanning/scanning-analysis-collector"`?

Reply:
You are right, thanks for noticing. The README is wrong. The other configmaps are also adding the protocol:

- https://github.com/sysdiglabs/charts/blob/232eb059c9a994c5538b283b9207de98eeec9e4b/charts/sysdig/templates/configmap-benchmark-runner.yaml#L10
- https://github.com/sysdiglabs/charts/blob/232eb059c9a994c5538b283b9207de98eeec9e4b/charts/sysdig/templates/configmap-host-analyzer.yaml#L10
- 
So I think in the example we should provide just the address without the protocol